src/cli_plugin/install/zip.test.js

```diff
@@ -53,18 +53,14 @@ describe('kibana cli', function () {
         const archive = path.resolve(repliesPath, 'test_plugin.zip');
         await extractArchive(archive, tempPath, 'kibana/test-plugin');
 
-        expect(glob.sync('**/*', { cwd: tempPath })).toMatchInlineSnapshot(`
+        expect(globby.sync('**/*', { cwd: tempPath })).toMatchInlineSnapshot(`
           Array [
-            "bin",
+            "kibana.json",
             "bin/executable",
             "bin/not-executable",
-            "kibana.json",
-            "node_modules",
-            "node_modules/some-package",
+            "public/index.js",
             "node_modules/some-package/index.js",
             "node_modules/some-package/package.json",
-            "public",
-            "public/index.js",
           ]
```

Comment:
I'm concerned by this snapshot change

Reply:
Pushed https://github.com/elastic/kibana/pull/138571/commits/ce87e594c64a210195472ca8a32d4fdb7671adff.  It's only used in tests.

Reply:
Hmm, what's only used in tests? I'm wondering why we're not `onlyFiles: false` to all existing uses of `glob` to maintain consistency?

Reply:
I went through each replacement and added onlyFiles: false to all uses that are

1) generic exported functions
2) matching on paths without extensions, anything ending with a `*`

https://github.com/elastic/kibana/pull/138571/commits/eb945900b3b4fbf14aa06f6cda1788a11b3ad6e4 + https://github.com/elastic/kibana/pull/138571/commits/49c4edf133f963f18e97e07a558ca9f2e3aee96c

I can update the remaining, but I think we should be okay.  *.ts technically could match a folder, but I expect other globs would be broken too.  Either way works for me.

Reply:
Sounds great, thank you for being thorough!